SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs

```diff
@@ -55,6 +55,13 @@ shift 1
 
         public const string SimulatedPlayerCoordinatorShellScript =
 @"#!/bin/sh
+
+# Some clients are quite large so in order to avoid running out of disk space on the node we attempt to delete the zip
+WORKER_ZIP_DIR=""/tmp/runner_source/""
+if [ -d ""$WORKER_ZIP_DIR"" ]; then
```

Comment:
```suggestion
if [[ -d ""${WORKER_ZIP_DIR}"" ]]; then
```

Reply:
Extended syntax not needed for testing if a directory exists. What I've written is available on all POSIX shells so I prefer to stick with it as long as the condition doesn't require more.

Reply:
Side note:  this check isn't really required because `rm -rf`